v3.9/reference/typha/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Typha Overview
+redirect_from: latest/reference/typha/index
+canonical_url: 'https://docs.projectcalico.org/v3.9/reference/typha/index'
+---
+
+The Typha daemon sits between the datastore (such as the Kubernetes API server) and many instances of Felix. Typha's main purpose is to increase scale by reducing each node's impact on the datastore.  Services such as [Felix](https://github.com/projectcalico/felix) and [confd](https://github.com/projectcalico/confd) connect to Typha instead of connecting directly to the datastore as Typha maintains a single datastore connection on behalf of all its clients. It caches the datastore state and dedupes events so it can be fanned out to many listeners.
+
+> **Note**: It is recommend to only use Typha if you're using the Kubernetes API Datastore and you have more than 50 Kubernetes nodes. While Typha can be used with etcd, etcd v3 is optimised to handle many clients already so we do not recommend adding Typha if you're using etcd.
```

Comment:
The first sentence kind of sounds like you shouldn't use Typha if you have less than 50 nodes, I don't think we should recommend against that as it is fine to do.
Maybe it should be something like "When using the Kuberenetes API Datastore, Typha can be used and is recommended when using more than 50 Kubernetes nodes."

Reply:
And, FWIW, we're intending to make Typha default enabled for all Kubernetes datastore deployements, so we probably can just get rid of this note altogether?

Reply:
I have updated the note based on @bmckercher123's comments, so I'll leave the decision up to you if you think it should be removed altogether.